libs/utils.py

```diff
@@ -49,14 +54,18 @@ def cleanup_after_test(workload):
         kill_process_by_name("secret_prov_server_dcap")
         kill_process_by_name("/gramine/app_files/apploader.sh")
         kill_process_by_name("/gramine/app_files/entrypoint")
-        utils.run_subprocess('sudo sh -c "echo 3 > /proc/sys/vm/drop_caches"')
-        utils.run_subprocess("docker rmi gsc-{}x -f".format(workload))
-        utils.run_subprocess("docker rmi gsc-{}x-unsigned -f".format(workload))
-        utils.run_subprocess("docker rmi {}x -f".format(workload))
-        utils.run_subprocess("docker rmi verifier_image:latest -f")
-        utils.run_subprocess("docker system prune -f")
+        kill_process_by_name("/gramine/meson_build_output/lib/x86_64-linux-gnu/gramine/sgx/loader")
+        run_subprocess('sudo sh -c "echo 3 > /proc/sys/vm/drop_caches"')
+        run_subprocess("docker rmi gsc-{}x -f".format(workload))
```

Comment:
Do we need these kill commands anymore if they are getting removed by the app itself?

Reply:
These images are not removed by app itself. These are final signed images which we have to remove